app/internal/languages.py

```diff
@@ -0,0 +1,85 @@
+import glob
+import json
+import pathlib
+from typing import Dict, Union
+
+from app import config
+
+LANGUAGE_FILES_PATH = "app/languages/*.json"
+LANGUAGE_FILES_PATH_TEST = "../app/languages/*.json"
+
+translations_dict = {}
+
+
+def get_translations_dict() -> Dict[str, Union[str, Dict[str, str]]]:
+    """Gets and returns the translations_dict, which is a dictionary of
+    the translated words in either the user's language setting,
+    or the default app setting.
+
+    Returns:
+        dict[str, Union[str, Dict[str, str]]]: a dictionary of string keys and
+        their translation as their values. The value can either be a string,
+        or a nested dictionary for plural translations.
+    """
+    if translations_dict:
+        return translations_dict
+    # TODO: Waiting for user registration. Restore when done.
+    # display_language = get_display_language(user_id)
+    # update_translations_dict(display_language)
+    update_translations_dict(config.WEBSITE_LANGUAGE)
```

Comment:
Maybe:

```
translations = populate_with_language(config.WEBSITE_LANGUAGE)
```

This way, the reader gets more information about what happens, and the functions remain pure and easily testable.